### DISCUSSION TITLE
Way to retain original mastery position in the event of revision and remaster

### DISCUSSION THREAD

**Discussion by suspect15** [RetroAchievements/RAWeb#1240] [Ideas]
A few people have recently expressed to me some pretty big disappointment having lost their original mastery position rank due to minor revisions that occurred.  I understand why people who put in the effort to master a set first, then lose that position unexpectedly due to a small revision that is later made feel very dejected.  I'm wondering if there would be an easy way for someone to retain their original ranking after they remaster a revision.

For example, the first person to master a set will jump back to the first mastery after a revision is released upon remastering the set.

I realize this is a bit of a sticky discussion and there are a lot of potential counterarguments, but I think the most harm is done to the people that put in significant effort early in a set's release to secure either a first mastery or first 10 spot.  That's why I'm suggesting this feature be added.

**Comment by suXinjke** [comment]
If mastery badges are retained then one could rely on counting these?

**Comment by redwizard42** [comment]
What about multiple mastery order lists. Once a revision is made store the original masteries as a "rev 1" and then there's historical tracking. I mean... the DB implementation is almost certainly not straight forward, but that gives the best of both worlds.
